pkg/core/resources/apis/mesh/traffic_log_validator.go

```diff
@@ -0,0 +1,42 @@
+package mesh
+
+import (
+	"fmt"
+
+	mesh_proto "github.com/Kong/kuma/api/mesh/v1alpha1"
+	"github.com/Kong/kuma/pkg/core/validators"
+)
+
+func (d *TrafficLogResource) Validate() error {
+	var err validators.ValidationError
+	err.Add(d.validateSources())
+	err.Add(d.validateDestinations())
+	return err.OrNil()
```

Comment:
Why don't we validate `conf` ?
What is the value in a `TrafficLog` without a destination for logs ?